flow/designs/asap7/mock-array/power.tcl

```diff
@@ -20,7 +20,7 @@ for {set x 0} {$x < 8} {incr x} {
 
 report_parasitic_annotation
 report_power
-read_power_activities -scope TOP/MockArray -vcd designs/src/mock-array/MockArrayTestbench.vcd
+read_power_activities -scope TOP/MockArray -vcd $::env(RESULTS_DIR)/MockArrayTestbench.vcd
```

Comment:
@maliberty @jeffng-or What is the difference between the RESULTS_DIR and REPORTS_DIR? What goes where?

.lib, .lef, .odb, .sdc seems obviously "results". Less sure about .vcd.

Arbitrary? I think we should minimize surprises. I'm somewhat surprised to see a .vcd file as a "result" and not a "report". It is somewhat like a "log", but perhaps more like a "report" since it is structured very specifically.

Reply:
Its a bit arbitrary but I think of a report as something a human would inspect in some way.  I doubt anyone wants to read a vcd file.  A log is more of a record of the tools output while running.

Reply:
a .rpt file of many megabytes is best viewed in OpenROAD GUI

Reply:
That is still human inspection.  There is no GUI for a vcd file and I don't see one that would make sense.  It is just inter-tool data.

Reply:
Sure there is a GUI: gtkwave

Reply:
Fair point - I don't really care that much.  If you want to PR a change to report that's fine.

Reply:
I'm just collecting information for now, no strong opinion, just curious how we ended up with two and if we had a clear objective selection criterea. It is a bit fuzzy, that's fine.

Reply:
It predates me so I'm just giving my opinion on how it might make sense.